lib/workers/global/config/parse/file.ts

```diff
@@ -45,6 +45,12 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
     if (err instanceof SyntaxError || err instanceof TypeError) {
       logger.fatal(`Could not parse config file \n ${err.stack}`);
       process.exit(1);
+    } else if (err instanceof ReferenceError) {
```

Comment:
I think we should check if the file exists earlier. Remember that the file is optional

Reply:
Hi @viceice I've reviewed the issue description another time and also tried to take your suggestion into account. So I came to this solution:

1. Consider the config file optional and don't complaint/exit if it doesn't present.
2. If `enc.RENOVATE_CONFIG_FILE` is specified, check if that file exists and exit with fatal error if it desn't exist.
3. When reading JS config file, check for `ReferenceError` and exit with fatal error if some varialbe can't be resolved.